inc/stat.class.php

```diff
@@ -635,20 +635,26 @@ static function showTable($itemtype, $type, $date1, $date2, $start, array $value
             }
             echo Search::showItem($output_type, $timedisplay, $item_num, $row_num);
 
+            //le nombre d'interventions résolues ayant une durée
```

Comment:
only in english please ;)

Reply:
Ok. I just copy / paste code with french comments